.travis.yml

```diff
@@ -4,6 +4,8 @@ sudo: required
 go:
   - 1.8.x
   - 1.9.x
+  - 1.10.x
+  - 1.x
```

Comment:
what does `1.x` do? And I would think we can drop `1.8.x`

Reply:
1.x runs on the latest `1.x` line. Really it is currently redundant with 1.10.x, but will work when 1.11 is released.